packages/mail-client/src/mail-client-types.ts

```diff
@@ -333,4 +333,8 @@ export interface SmtpTransportOptions {
    * Connected socket to use instead of creating and connecting a new one. If secure option is true, then socket is upgraded from plaintext to ciphertext.
    */
   connection?: net.Socket | undefined;
+  /**
+   * A proxy URL used for connecting the SMTP server.
```

Comment:
[pp] Perhaps be explicit that this string is handed to the nodemailer implementation which does the proxy handling for us.